Headers/Foundation/NSHTTPCookie.h

```diff
@@ -204,13 +204,13 @@ extern NSString * const NSHTTPCookieVersion; /** Obtain cookie version */
  * Returns whether the receiver should only be sent over
  * secure connections.
  */
-- (BOOL) isSecure;
+@property (readonly, getter=isSecure) BOOL secure;
 
 /**
  * Returns whether the receiver should be destroyed at the end of the
  * session.
  */
-- (BOOL) isSessionOnly;
+@property (readonly, getter=isSessionOnly) BOOL sessionOnly;
```

Comment:
To clarify: if the `@property` syntax is required for compatibility, please provide it as an alternative and guarded by appropriate feature guards.

Alternatively, provide `- (BOOL) isSessionOnly` plus `- (BOOL) sessionOnly` as a wrapper around it.

`@property` is really breaking the 3.4.5 requirement which we outlined in the mailing list.